js/src/logger.ts

```diff
@@ -1198,13 +1126,14 @@ type InitDatasetOptions = {
  * @param options.apiKey The API key to use. If the parameter is not specified, will try to use the `BRAINTRUST_API_KEY` environment variable. If no API
  * key is specified, will prompt the user to login.
  * @param options.orgName (Optional) The name of a specific organization to connect to. This is useful if you belong to multiple.
+ * @param options.outputInsteadOfExpected Unless set to false, records will be fetched from this dataset in the legacy record format, with "output" as an alias for the "expected" field. This will default to false in a future version of Braintrust.
```

Comment:
"as an alias" kind of makes it sound like we'll have both "output" and "expected". Maybe `with the "expected" field renamed to "output"`?